src/controllers/confirm.company.controller.ts

```diff
@@ -91,7 +91,7 @@ const stopPages = {
   [EligibilityStatusCode.INVALID_COMPANY_STATUS]: Templates.INVALID_COMPANY_STATUS,
   [EligibilityStatusCode.INVALID_COMPANY_TRADED_STATUS_USE_WEBFILING]: Templates.USE_WEBFILING,
   [EligibilityStatusCode.INVALID_COMPANY_TYPE_USE_WEB_FILING]: Templates.USE_WEBFILING,
-  [EligibilityStatusCode.INVALID_COMPANY_APPOINTMENTS_MORE_THAN_ONE_OFFICER]: Templates.USE_WEBFILING,
+  [EligibilityStatusCode.INVALID_COMPANY_APPOINTMENTS_INVALID_NUMBER_OF_OFFICERS]: Templates.USE_WEBFILING,
```

Comment:
Just noticed that there doesn't seem to be a test for the invalid company appointments. Could you add an extra test into confirm.company.controller.unit.ts to check the USE_WEBFILING screen is shown when INVALID_COMPANY_APPOINTMENTS_INVALID_NUMBER_OF_OFFICERS is returned from api? Should be very similar to other tests in there for the other stop screens.